src/POD.jl

```diff
@@ -1,29 +1,41 @@
+__precompile__()
+
 module POD
 
+PODDEBUG = false
+
 using JuMP
 using MathProgBase
 using Compat
 
+PODDEBUG && using Gurobi
+
+include("const.jl")
+
 # Engine for High-level Algorithmic Control and User-interface
-include("algorithm.jl")
 include("solver.jl")
+include("mpb2moi.jl") # Transition file
```

Comment:
I am sort of in two minds here - whether to keep an moi implementation as a separate repo like the other solver repos or integrate it. We can discuss with russell @rb004f

Reply:
Yeah. I think depends on how many things we are using to interact with either MOI or MPB, which are all collected in this file.